cfme/services/dashboard/ssui.py

```diff
@@ -66,6 +82,10 @@ def total_services(self):
     view = navigate_to(self, 'TotalServices')
     view.flash.assert_no_error()
     view = self.create_view(MyServicesView)
+    wait_for(
```

Comment:
@mfalesni should we also create a general helper for creating views that are visible after return,
i notice this pattern is going to repeat more and more

Reply:
I think that would be a workaround for different issues. We now support waiting for views to appear in navigate_to ... and when you are creating a view, you assume it already is displayed.

Reply:
good point, i wonder if this a issue unique to ssui, or if this just needs an added navigation step

Reply:
In Tuesday's team meeting @psav mentioned including wait_for(view.is_displayed) in framework code .So I guess this will go away soon.